### PR TITLE
Increase integration test coverage on CRIO environmemnt on main branch

### PR DESCRIPTION
**Pull request by xiancao**
Increase integration test coverage on CRIO environmemnt on main branch

Jenkins result:
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11625/ (kind sequential)
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/11626/ (kind parallel)
https://build.weblogick8s.org:8443/job/wko-olcne-main-nightly/20/ (olcne)
https://build.weblogick8s.org:8443/job/wko-olcne-main-nightly/22/ (olcne parallel)

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -562,6 +562,13 @@
                 <groups>crio</groups>
             </properties>
         </profile>
+        <profile>
+            <id>olcne-cert</id>
```
Comment (anpanigr) [inline integration-tests/pom.xml:566]: id should match the tag as we do for other profile

**Review by anpanigr — COMMENTED** [review]
Can you run the test in parallel.

**Review by anpanigr — APPROVED** [review]
Please sync to the main

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**